File: ABC/141/C.cpp
#include <iostream>
#include <string>

using namespace std;

int main(){

    int n,k,q;
    int a[q];
    int p[n];

    //input
    cin >> n >> k >> q;
    for(int i = 0; i < q; i++)
    {
      cin >> a[i];
    }

    for(int i = 0; i < n; i++)
    {
      p[i] = 1-k;
    }


    for(int i = 0; i < q;i++)
    {
      p[a[i]-1] = p[a[i]-1] + 1;
    }

    for(int i = 0; i < n;i++)
    {
      printf("%d\n",p[i]);
      if(p[i] > 0)
      {
        printf("Yes\n");
      }
      else
      {
        printf("No\n");
      }
    }


    return 0;
}

File: ABC/141/C.py
n,k,q = map(int,input().split())
a_n = [int(input()) for i in range(q)]

p_n = [k-q for i in range(n)]

for a_i in a_n:
    p_n[a_i-1] += 1

[print("Yes") if p_i > 0 else print("No") for p_i in p_n]
